neonvm-kernel/linux-config-aarch64-6.12.24

```diff
@@ -628,33 +629,12 @@ CONFIG_ACPI_PCC=y
 CONFIG_PMIC_OPREGION=y
 CONFIG_ACPI_VIOT=y
 CONFIG_ACPI_PRMT=y
-CONFIG_IRQ_BYPASS_MANAGER=y
-CONFIG_HAVE_KVM=y
```

Comment:
I was initially slightly concerned about this, but looks like the removal is expected.

More info here: https://lore.kernel.org/lkml/20240108124740.114453-6-pbonzini@redhat.com/T/

Reply:
I also don't see any configurable option for arm64 🤔 
https://www.kernelconfig.io/search?arch=arm64&kernelversion=6.12.24&q=CONFIG_KVM&page=0